packages/ag-charts-website/src/features/api-documentation/components/ApiReference.tsx

```diff
@@ -29,6 +29,7 @@ import {
 } from '../apiReferenceHelpers';
 import { SelectionContext } from './OptionsNavigation';
 import { type CollapsibleType, PropertyTitle, PropertyType } from './Properties';
+import { ChildPropertiesButton } from './Properties';
```

Comment:
Since `ChildPropertiesButton` is only used in this file, you could move it in here, and rename `CollapsibleButton` in `Properties.tsx` to `CodeCollapsibleButton`